editor/app/src/PythonExecutor.cs

```diff
@@ -114,15 +114,53 @@ internal class PythonExecutor {
             }
         }
 
-        private const string PYTHON_INTERPRETER = "python";
```

Comment:
Thanks @jonsimantov for fixing this!
Can you please update when is this expected to be released?